dv/formal/ibex_formal.core

```diff
@@ -10,19 +10,9 @@ filesets:
     depend:
       - lowrisc:ibex:ibex_pkg
       - lowrisc:ibex:ibex_core
-      - lowrisc:prim:buf
-      - lowrisc:prim:clock_gating
-      - lowrisc:prim:secded
-      - lowrisc:prim:assert
-      - lowrisc:prim:ram_1p_pkg
-      - lowrisc:prim_generic:buf
-      - lowrisc:prim_generic:clock_gating
-    files:
-      - ../../rtl/ibex_register_file_ff.sv # generic FF-based
-      - ../../rtl/ibex_register_file_fpga.sv # FPGA
-      - ../../rtl/ibex_register_file_latch.sv # ASIC
-      - ../../rtl/ibex_top.sv
-    file_type: systemVerilogSource
+      - lowrisc:ibex:ibex_top
```

Comment:
Have you been able to try this out? I'm just wondering whether the explicit dependencies were to allow the formal run to look at just a subset.

The existing structure is a rather ugly way to do that, but this cleanup might break things?

Reply:
I ran the formal flow and so does the CI, but this was my first time running it, so I don't know whether this already covers everything. But the basic formal flow works.